### DISCUSSION TITLE
For a given SyncTarget, how can I determine which workspaces have content going to it?

### DISCUSSION THREAD

**Discussion by ncdc** [kcp-dev/kcp#1966] [General]
Split out from https://github.com/kcp-dev/kcp/issues/1909

@cmoulliard could you please continue the conversation for this topic here? What use cases are you trying to solve? Thanks!

**Comment by cmoulliard** [comment]
As an Ops engineer operating on a kcp+k8s platform, I will be interested to figure out easily which workspaces are `sync` to a k8s cluster in order to report problems to the team/user working top of the workspace where error(s) have been discovered
